### PR TITLE
feat: add new language (lt-LT)

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

File: locales/lt-LT/messages.po [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+msgid "Unable to extract the archive"
+msgstr "Nepavyko išarchyvuoti archyvo"
+
+msgid "Upload Media"
+msgstr "Įkelti Mediją"
+
+msgid "Upload new media"
+msgstr "Įkelti naują mediją"
+
+msgid "You can\'t upload a zip file as a media"
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: As I remember it will not work if you use backslashes

File: locales/lt-LT/messages.po [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"X-Poedit-Basepath: ../../\n"
+"X-Poedit-KeywordsList: __\n"
+"X-Poedit-SearchPath-0: .\n"
+
+msgid "Add a media file"
+msgstr "Pridėti medijos failą"
+
+msgid "An error has occurred. Please contact your administrator."
+msgstr ""
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: without a translation

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: it's normal

File: locales/lt-LT/messages.po (hunk 1 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

**Pull request merged**